mailpoet/lib/WooCommerce/Emails.php

```
@@ -0,0 +1,100 @@
+  }
+
+  /**
+   * Locate block templates for MailPoet emails.
+   *
+   * @param string $template The template path.
+   * @param string $template_name The template name.
+   * @param array $args The template arguments.
+   * @param string $template_path The template path.
+   * @return string The located template path.
+   */
+  public function locateBlockTemplate($template, $template_name, $args, $template_path) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)